test/core/annotations.wast

```diff
@@ -25,6 +27,7 @@
 (assert_malformed (module quote "(@a \07)") "illegal character")
 (assert_malformed (module quote "(@a \08)") "illegal character")
 (module quote "(@a \09)")  ;; \t
+(;
```

Comment:
I don't remember why this was commented out. Can you reactivate and check if anything goes wrong?

Reply:
all good, I will remove the comments